tools/kshttp-pyhton/vdivde.py

```diff
@@ -0,0 +1,57 @@
+"""
+Example for using the kshttp-python functions
+In this example we change the times of videvde3696 blocks to account for different behaviour when switching to a
+build with fixed tasklist patch.
+This only works if all relevant vdivde3696 blocks run with the cycle time of the UrTask.
+"""
+
+from kshttp import *
+
+
+def fix_vdivde(server_name: str = 'MANAGER', host: str = 'localhost', max_time: float = 30):
+    """Fix some vdivde3696 blocks with new build
+
+    :param str server_name: name of server to fix
+    :param str host: host of server
+    :param float max_time: maximal time value of vdivde3696 blocks to decide if they need fixing
+    :return: None
+    """
+    s = server_by_name(server_name, host=host)
+
+    # fix vdivde3696 timer
+    if object_exists(s, '/acplt/vdivde3696'):
```

Comment:
Do not hard code that path.
  You can iterate over all libraries instantiations of
  `/acplt/ov/library` (this one is really fixed)

https://github.com/acplt/rte/blob/02f9eb1e42a1a42216e312fd971653a9f9e7f46e/addonlibs/hmi/hmiJavaScript/JavaScript/hmi-class-HMIJavaScriptKSClient.js#L1067

Reply:
Since this is only a script for quick-fixing some of the real-life issues resulting from the fix in #60, I consider it okay that this script takes the short cut, which may not be 100% correct but sufficient in most cases. It should be used with care anyhow.

Reply:
Than at least add a comment.